src/main/java/gg/uhc/uhc/modules/portals/EndModule.java

```diff
@@ -86,17 +88,19 @@ public void onDisable() {
         }
     }
 
-    @EventHandler(ignoreCancelled = true)
+    @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
```

Comment:
Good argument, a well behaved multiworld plugin should be listening on LOW/LOWEST. Maybe it's possible to dynamically change this based on configuration and manually registering into the HandlerList, but for now I'll just change this to HIGH.